src/components/TuneLabelInput.vue

```diff
@@ -3,12 +3,18 @@ import TuneIconHint from './TuneIconHint.vue';
 
 defineProps<{
   hint?: string;
+  sublabel?: string;
 }>();
 </script>
 
 <template>
-  <div class="tune-label mb-[2px] flex items-center gap-1">
-    <slot />
-    <TuneIconHint :hint="hint" />
+  <div class="tune-label-container flex flex-col">
+    <div class="tune-label mb-[2px] flex items-center gap-1">
+      <slot />
+      <TuneIconHint :hint="hint" />
+    </div>
+    <div v-if="sublabel" class="tune-sublabel text-sm text-skin-text opacity-60">
```

Comment:
Can you add the `tune-sublabel` class to the styles.scss file and move `text-sm text-skin-text opacity-60` into that?